test/describe_context.rb

```diff
@@ -0,0 +1,9 @@
+BDDSM.describe 'Test case with context' do
+  context 'Context example 1' do
```

Comment:
Is it possible to add tests to check error messages generated when such test failed?

Reply:
Not sure about you are meaning. Maybe you can give me some example of this?